pkg/cmd/apply/options.go

```diff
@@ -98,7 +99,9 @@ func (o *Options) Run() error {
 	}
 
 	// Compute changes for preview
-	changes, err := previewcmd.Preview(&o.Options, stateStorage, sp, project, stack)
+	// changes, err := previewcmd.Preview(&o.Options, stateStorage, sp, project, stack)
+	previewOptions := engineapi.NewAPIOptions()
```

Comment:
I see the `engineapi.NewAPIOptions() != &o.Options`

Reply:
Right. `o.Options` is the command line options for the preview command, it's different from `engineapi.NewAPIOptions()` which creates a new engine api option (current it looks like a subset of the command line options).